sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -42,6 +42,10 @@ import scalajsbundler._
   *
   * Version of webpack-dev-server to use.
   *
+  * {{{
+  *   version in webpack := "2.7.1"
```

Comment:
typo: should be `version in installWebpackDevServer` (if we keep using this key)